src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -156,8 +156,8 @@
             *ngIf="permissions.prometheus.read">
           <a routerLink="/monitoring">
             <ng-container i18n>Monitoring</ng-container>
-            <small *ngIf="prometheusAlertService.alerts.length > 0"
-                   class="badge badge-danger">{{ prometheusAlertService.alerts.length }}</small>
+            <small *ngIf="prometheusAlertService.activeAlerts > 0"
+                   class="badge badge-danger">{{ prometheusAlertService.activeAlerts }}</small>
```

Comment:
I was curious about whether this could be achieved with just template changes and pipes, given that it's pure data presentation. I found [this library](https://www.npmjs.com/package/ngx-pipes#filterby):
```suggestion
            <small *ngIf="prometheusAlertService.alerts | filterBy: ['status']: 'active' as activeAlerts"
                   class="badge badge-danger">{{ activeAlerts.length }}</small>
```